Machine_Learning/project.py

```
@@ -0,0 +1,115 @@
+import streamlit as st
+import pickle
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)